universum/configuration_support.py

```diff
@@ -40,10 +40,20 @@ class Step:
         will not be processed correctly and therefore should be splat into ``["ls", "-a"]``. Lists like
         ``["build.sh", "--platform A"]`` will not be processed correctly and thus should be plat into
         ``["build.sh", "--platform", "A"]``. A build step can have an empty list as a command - such step won't do
-        anything except showing up the step name in Universum execution logs. Some common actions, such as ``echo``
-        or ``cp``, are bash features and not actual programs to run. These features should be called as
-        ``["bash", "-c", "echo -e 'Some line goes here'"]``. Note that in this case the string to be passed to bash
-        is one argument containing white spaces and therefore not splat by commas.
+        anything except showing up the step name in Universum execution logs. Some common actions, such as ``echo``,
+        are bash features and not actual programs to run. These features should be called as
+        ``["bash", "-c", "echo -e 'Some line goes here'"]``. Any other shell can also be used instead of bash.
+        Note that in this case the string to be passed to bash is one argument containing white spaces
+        and therefore not splat by commas.
+
+        .. note::
+
+            Direct process launching is not entirely identical to using shell. Arguments including
```

Comment:
May I suggest a clearer wording for the first sentence? E.g.
"Different parsing rules apply, when the process is launched directly instead of using shell", or
"Direct launch of the process has different parsing rules from using shell"